plugin/public/css/block.css

```diff
@@ -1,20 +1,22 @@
-.folder {
-  background-color: rgb(197, 218, 224);
-  font-weight: 500;
+.sgdd-block-settings-folder, .sgdd-block-settings-folder-up {
+	color: var(--wp-admin-theme-color);
+  	font-weight: bold;
 }
 
-.folder label {
-  color: #333333;
+.sgdd-block-settings-folder label::before {
+	content: "> ";
```

Comment:
Eh, I'm not a fan. Just color looks better I think.

Reply:
As you want - it may be also added in user CSS so it is no problem for me. Or maybe can we find some nice icon with folder?

Reply:
Well, an icon would be the best solution I think...

Reply:
I was doing some research and there is possibility to use WordPress icon: [https://developer.wordpress.org/resource/dashicons/#open-folder](url) so I modify code to use these dashicons.

Reply:
That's perfect, thanks.